lua/telescope/config/resolve.lua

```diff
@@ -88,8 +93,7 @@ That's the next step to scrolling.
 local get_default = require('telescope.utils').get_default
 
 local resolver = {}
-
-local _resolve_map = {
+local _simple_resolve_map = {
```

Comment:
this is probably easier to understand by just doing `_resolve_map = {}` and then each of the following with `_resolve_map[function() ...] = ...` etc.

I don't think we need to do the copy thing, right?

Or is this because only some keys are considered "simple"

Reply:
Originally, I was getting some issues where the `padding` case in `_resolve_map` was referring back to `_resolve_map`.

I don't seem to be getting those issues now, so I've changed to just using `_resolve_map` instead of `_simple_resolve_map`.
I'll keep an eye out for any issues re-emerging.